src/main/scala/nl/biopet/tools/fastqsplitter/ArgsParser.scala

```diff
@@ -11,10 +11,10 @@ class ArgsParser(cmdName: String) extends AbstractOptParser[Args](cmdName) {
     .valueName("<file>")
     .action((x, c) => c.copy(inputFile = x))
     .text("Path to input file")
-  opt[File]('o', "output")
+  opt[File]('o', "outputFile") // outputFiles?
```

Comment:
The argument is a single file, you can supply is multiple time with `-o file1 -o file2`